### PR TITLE
Add Alpine Linux (apk) support

### LINKED ISSUE #25: apk Alpine gonna be supported in futur ?
Alpine is very great, their package is update regularly, stable, installation is blazing fasssssttt compare to Ubuntu / other 😄

If you want to try how alpine is fast compare with ffmpeg

```
Alpine :

docker run -it --rm alpine

apk update && apk add ffmpeg

Ubuntu :

docker run -it --rm Ubuntu

apt update && apt install ffmpeg -y
```

Ubuntu package is old for almost all things, alpine have frequently updated version of packages. Ffmpeg is old on Ubuntu more than 2 years almost.

Sorry for my English 😅

### PR DESCRIPTION
**Pull request by PureTryOut**
Fixes #25.

We're looking into maybe using apx in postmarketOS (no promises!), adding Alpine Linux support would be a first step for that.

### CODE REVIEW

**Comment by kbdharun** [comment]
Wow, this is huge 🔥 .

**Comment by mirkobrombin** [comment]
Thanks for your contribution. I'll review soon.

File: cmd/clean.go [hunk 1]
```diff
@@ -44,12 +44,15 @@ func NewCleanCommand() *cobra.Command {
 func clean(cmd *cobra.Command, args []string) error {
 	aur := cmd.Flag("aur").Value.String() == "true"
 	dnf := cmd.Flag("dnf").Value.String() == "true"
+	apk := cmd.Flag("dnf").Value.String() == "true"
```
Comment (kbdharun) [inline cmd/clean.go:47]:
```suggestion
	apk := cmd.Flag("apk").Value.String() == "true"
```
Shouldn't this be apk?

File: cmd/initialize.go [hunk 2]
```diff
@@ -44,12 +44,15 @@ func NewInitializeCommand() *cobra.Command {
 func initialize(cmd *cobra.Command, args []string) error {
 	aur := cmd.Flag("aur").Value.String() == "true"
 	dnf := cmd.Flag("dnf").Value.String() == "true"
+	apk := cmd.Flag("dnf").Value.String() == "true"
```
Comment (kbdharun) [inline cmd/initialize.go:47]:
```suggestion
	apk := cmd.Flag("apk").Value.String() == "true"
```

File: cmd/list.go [hunk 3]
```diff
@@ -48,12 +48,15 @@ func NewListCommand() *cobra.Command {
 func list(cmd *cobra.Command, args []string) error {
 	aur := cmd.Flag("aur").Value.String() == "true"
 	dnf := cmd.Flag("dnf").Value.String() == "true"
+	apk := cmd.Flag("dnf").Value.String() == "true"
```
Comment (kbdharun) [inline cmd/list.go:51]:
```suggestion
	apk := cmd.Flag("apk").Value.String() == "true"
```

**Review by kbdharun — APPROVED** [review]
Other than the requested changes, the PR LGTM. Thanks for your contribution btw. Nice to see new options added to `apx`.

**Comment by PureTryOut** [comment]
That's what you get for copy-pasting mindlessly. That common piece of code needs to be refactored to it's own function really, it's repeated way too often. In a separate PR :shrug:

**Comment by kbdharun** [comment]
> That's what you get for copy-pasting mindlessly. That common piece of code needs to be refactored to it's own function really, it's repeated way too often. In a separate PR shrug

I guess we can optimise it in future.

**Comment by TheNexter** [comment]
So happy and excited to get this in VanillaOS with APX 😁

**Pull request merged**